App.js

```diff
@@ -1,26 +1,36 @@
-import React from 'react';
-import styled from 'styled-components/native';
+import React from "react";
+import { ImageBackground, StyleSheet } from "react-native";
```

Comment:
I love all the thing s you can download to make react native (and react) work smoother. Like the ImageBackground